docs/pages/use-teleport/teleport-connect.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Using Teleport Connect
+description: Using Teleport Connect
+---
+
+Teleport Connect provides easy and secure access to SSH servers and databases, with support for
+other resources such as Kubernetes clusters and application access coming in the future.
+
+![resources tab in Teleport Connect](../../img/use-teleport/connect-cluster.png)
+
+## Installation & Upgrade
```

Comment:
This and other headings should be in sentence case, with only the first letter and any proper nouns capititalized

Reply:
> This and other headings should be in sentence case, with only the first letter and any proper nouns capititalized

Thanks! I wasn't sure and I usually saw most of my US-based peers use title case for those.